test/tall/invocation/chain_postfix.stmt

```diff
@@ -47,7 +47,6 @@ someReceiverObject.property1.property2
     .property5
     .property6;
 <<<
-### TODO(tall): Allow splitting between successive indexes.
```

Comment:
I think this just isn't worth doing. It never looks good and users are better off either reorganizing their code or allowing it to run long.